mkeditorpacks/README.md

```diff
@@ -0,0 +1,19 @@
+MkEditorPacks
+=============
+
+This a tool suite to build game packs for map editors.
+
+It is developped for the [Unvanquished](https://unvanquished.net/) project, but must fit any project supported by the editors listed below.
+
+Supported editors
+-----------------
+
+- [GtkRadiant](http://icculus.org/gtkradiant/)
+- [NetRadiant](https://gitlab.com/xonotic/netradiant)
+
+Authors
```

Comment:
Hey, I missed that comment! So what I have to do? add a COPYING file with BSD 3-clause inside and some headers to python files? I'm OK with `Unvanquished Development`, there is names in git history. Is years 2014-2017 OK?